vm/src/builtins.rs

```diff
@@ -559,11 +559,22 @@ fn builtin_pow(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+// Idea: Should we have a 'default' attribute, so we don't have to use OptionalArg's in this case
```

Comment:
I was thinking the same - in that case maybe we should tweak the design a bit to avoid multiple attributes (and conflicts w/ other proc-macros that use `#[default]`) to something like `#[pyarg(positional, default = "<expr>")]`.

Reply:
That does seem better syntax, with the advantage of being only one attribute.